PF/Assignment1/assignment6.java

```diff
@@ -0,0 +1,62 @@
+package programing_fundamental;
+import java.util.*;
+
+public class assignment6 {
+	public int[] join(int a[], int asize, int b[], int bsize, int c[])
+	{
+		int i,j,l;
+		int k=0;
+		for(i=0,j=0;i<asize&&j<bsize;i++,j++)
```

Comment:
ACTION: Instead if increasing variable i and j  every time and decreasing them conditionally , Increase them conditionally then u don't need to decrease them.